Multiple include of /spin1/users/browna6/pwgs_snakemake/helpers.py ignored
UnicodeDecodeError in line 7 of /spin1/users/browna6/pwgs_snakemake/phyloWGSparsing.snakefile:
'utf-8' codec can't decode byte 0xca in position 111: unexpected end of data
  File "/spin1/users/browna6/pwgs_snakemake/run_full_pipeline.snakefile", line 5, in <module>
  File "/spin1/users/browna6/pwgs_snakemake/phyloWGSparsing.snakefile", line 7, in <module>
  File "/usr/local/Anaconda/envs_app/snakemake/5.4.0/lib/python3.6/site-packages/pandas/io/parsers.py", line 678, in parser_f
  File "/usr/local/Anaconda/envs_app/snakemake/5.4.0/lib/python3.6/site-packages/pandas/io/parsers.py", line 446, in _read
  File "/usr/local/Anaconda/envs_app/snakemake/5.4.0/lib/python3.6/site-packages/pandas/io/parsers.py", line 1036, in read
  File "/usr/local/Anaconda/envs_app/snakemake/5.4.0/lib/python3.6/site-packages/pandas/io/parsers.py", line 1848, in read
  File "pandas/_libs/parsers.pyx", line 876, in pandas._libs.parsers.TextReader.read
  File "pandas/_libs/parsers.pyx", line 891, in pandas._libs.parsers.TextReader._read_low_memory
  File "pandas/_libs/parsers.pyx", line 968, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 1094, in pandas._libs.parsers.TextReader._convert_column_data
  File "pandas/_libs/parsers.pyx", line 1141, in pandas._libs.parsers.TextReader._convert_tokens
  File "pandas/_libs/parsers.pyx", line 1240, in pandas._libs.parsers.TextReader._convert_with_dtype
  File "pandas/_libs/parsers.pyx", line 1256, in pandas._libs.parsers.TextReader._string_convert
  File "pandas/_libs/parsers.pyx", line 1494, in pandas._libs.parsers._string_box_utf8
